docs/src/topics/get-started-with-clusterctl.md

```diff
@@ -34,10 +34,11 @@ Check cluster is ready:
 ```
 # Install clusterctl
 :warning: In order to install tools (clusterctl, ...) with makefile, you need to have installed golang to download binaries [golang][golang]
-You can install clusterctl for linux with:
+You can install  clusterctl for linux with:
 ```bash
   make install-clusterctl
 ```
+Or you can install clustercl with the cluterctl install section ([cluster-api][cluster-api]).
```

Comment:
cluterctl  (typo) 
you can write it better to not use install twice in the same sentence

Reply:
Done